nixos/modules/config/pulseaudio.nix

```diff
@@ -87,7 +87,8 @@ in {
     hardware.pulseaudio = {
       enable = mkOption {
         type = types.bool;
-        default = false;
+        default = config.sound.enable;
```

Comment:
It's IMHO better to set `hardware.pulseaudio.enable` in the `config` attribute of this module with `lib.mkDefault`. Otherwise the manual will be regenerated if the user sets `sound.enable` to false.

Reply:
OK, sure. I will need to do some more testing once my computer is done with some bigger rebuilds.